### PR TITLE
Add details about configuring g/w HA

### PR DESCRIPTION
**Pull request by aswinsuryan**
Signed-off-by: Aswin Surayanarayanan <asuryana@redhat.com>

### CODE REVIEW

File: src/resources/shared/openshift/run_prep_for_subm.md [hunk 1]
```diff
@@ -13,3 +13,10 @@ export IPSEC_NATT_PORT=4501
 export IPSEC_IKE_PORT=501
 export GW_INSTANCE_TYPE=m4.xlarge
 ```
+
+Submariner gateway can deployed in HA mode by setting the ENABLE_HA flag.
```
Comment (nyechiel) [inline src/resources/shared/openshift/run_prep_for_subm.md]:
```suggestion
Submariner gateway can be deployed in HA mode by setting the ENABLE_HA flag:
```

File: src/content/getting-started/quickstart/openshift/vsphere-aws/_index.md [hunk 2]
```diff
@@ -118,6 +118,12 @@ export IPSEC_NATT_PORT=4501
 export IPSEC_IKE_PORT=501
 ```
 
+* To deploy the gateway in HA mode set the ENABLE_HA flag
```
Comment (nyechiel) [inline src/content/getting-started/quickstart/openshift/vsphere-aws/_index.md]:
```suggestion
* To deploy the gateway in HA mode set the ENABLE_HA flag:
```

**Review by nyechiel — COMMENTED** [review]
Thanks @aswinsuryan. Can you please also update the in-repo README file? https://github.com/submariner-io/submariner/tree/devel/tools/openshift/ocp-ipi-aws

Also, there is a Markdown linting failure that needs to be addressed.

**Comment by aswinsuryan** [comment]
> Thanks @aswinsuryan. Can you please also update the in-repo README file? https://github.com/submariner-io/submariner/tree/devel/tools/openshift/ocp-ipi-aws
> 
> Also, there is a Markdown linting failure that needs to be addressed.

I have addressed the comments, will raise a PR for README.

**Review by tpantelis — APPROVED** [review]

**Review by nyechiel — APPROVED** [review]

**Pull request merged**